### PR TITLE
Readme and seeds

### PR DESCRIPTION
**Pull request by stellakunzang**
**Fixes issue #:** 

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Styling- no new features

**Description:**

Added actual seeds and removed faked data.

**What does this solve?:**

bye bye Bill Murray

**Notes:**

Also enhanced README.

- [x] All tests are passing

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -16,15 +16,15 @@ gem 'sass-rails'
 gem 'uglifier'
 gem 'coffee-rails'
 gem 'travis'
+gem 'faker'
+gem 'factory_bot_rails'
```
Comment (stellakunzang) [inline Gemfile:20]: We moved these out of the development test block so they would work in production

File: README.md [hunk 2]
```diff
@@ -2,10 +2,14 @@
...
 ## Introduction
 
-This application is a Ruby on Rails API service, utilizing GraphQL to create user-friendly endpoints. Our database is built using Postgres. You can find our schema in the section below.
+This application is a Ruby on Rails API service, utilizing GraphQL to create user-friendly endpoints. Our database is built using Postgres and you can find our schema in the section below.
+
+[Production link](https://streetwalker-backend.herokuapp.com/)
 
 ## Schema
 
+![Schema](/public/schema.png)
```
Comment (stellakunzang) [inline README.md:11]: We now have schema image.

**Review by stellakunzang — COMMENTED** [review]
I honestly don't know if putting photos in the public folder is smart or will work. It did make pushing the branch up take longer than usual, but this is the best way I know of to add images to the repo. If you think we should host them on cloudinary, I'm happy to adjust. Seemed like FE wanted real seeds with some sense of urgency.

**Comment by Gallup93** [comment]
Thanks for building out these seeds for us!

**Review by Gallup93 — APPROVED** [review]

**Review by elguapogordo — APPROVED** [review]

**Comment by elguapogordo** [comment]
photos in the app are more ideal than hotlinking but we've always been discouraged to host our own images. so if it works, cool. i could never figure it out

**Pull request merged**